connectors/source.py

```diff
@@ -398,6 +400,11 @@ def hash_id(cls, _id):
 
         return hash_id(_id)
 
+    @classmethod
```

Comment:
Will incremental syncs have a feature flag? Then we could guard the usage behind the feature flag removing the need for having a separate flag.

Reply:
Yes, there is one, see https://github.com/elastic/enterprise-search-team/issues/4629

The feature flag is at the connector level, and this is at the data source level (e.g. MySQL).

In 8.9, all SPO connectors will have the feature flag enabled, and SPO data source will have `support_incremental_sync` to return `True`